backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/webservice/FreinetApplicationMutationService.kt

```diff
@@ -23,8 +25,13 @@ import org.slf4j.LoggerFactory
 class FreinetApplicationMutationService {
     private val logger = LoggerFactory.getLogger(FreinetApplicationMutationService::class.java)
 
-    @GraphQLDescription("Send application info to Freinet")
-    fun sendApplicationDataToFreinet(applicationId: Int, project: String, dfe: DataFetchingEnvironment): Boolean {
+    @GraphQLDescription("Send application and card information to Freinet")
+    fun sendApplicationAndCardDataToFreinet(
+        applicationId: Int,
+        project: String,
+        freinetCards: List<FreinetCard>,
```

Comment:
❓why do we have list as param here? 
we can issue the card multiple times for one application, but we can't (and shouldn't) do it in one request, right?

Reply:
> It would actually work and there is no harm in doing it

the problem I see is that it's not obvious. we literally allow to enter different names for different cards on the form. how to the user should understand that all of them will be linked to the same user in freinet?

Reply:
Well but then we should not delete the query params immediately after forwarding from application. Because if there is any reload of the page the related applicant data will be lost and restriction to one card will not work